python/nano/src/bigdl/nano/pytorch/strategies/ddp_spawn.py

```diff
@@ -141,6 +140,10 @@ def launch(self, function: Callable, *args: Any,
 
         # restore the state of child process
         spawn_output = return_queue.get()
+
+        if trainer is None:
```

Comment:
Maybe adding in comments when the `trainer` will be `None`

Reply:
added it